recipes/alibi/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "alibi" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/alibi-{{ version }}.tar.gz
+  sha256: 20131a369871b32bc7abfe1caa47131657abc0e41c7f5c8e0d2cc6105a816a96
+
+build:
+  number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - attrs >=19.2.0,<22.0.0
+    - dill >=0.3.0,<0.4.0
+    - matplotlib-base >=3.0.0,<4.0.0
+    - numpy >=1.16.2,<2.0.0
+    - pandas >=0.23.3,<2.0.0
+    - pillow >=5.4.1,<9.0
+    - requests >=2.21.0,<3.0.0
+    - scikit-image >=0.14.2,!=0.17.1,<0.19
+    - scikit-learn >=0.20.2,<1.1.0
+    - scipy >=1.1.0,<2.0.0
+    - spacy >=2.0.0,<4.0.0
+    - tensorflow >=2.0.0,!=2.6.0,!=2.6.1,<2.8.0
+    - tqdm >=4.28.1,<5.0.0
+    - transformers >=4.7.0,<5.0.0
+    - typing-extensions >=3.7.2  # [py<38]
+
+test:
+  imports:
+    - alibi
+  commands:
+    - pip check
+  requires:
+    - pip
+    # Necessary for passing pip-check
+    - absl-py >=0.10.0,<0.11.0
```

Comment:
why are they not part of `run` if they are necessary to pass `pip check`?

Reply:
The build did not fail until the pip-check happened. So, am not clear on this part. Adding them to pip-requires, seems to make it pass without failure.

Reply:
Let me see what happens if I just move those under run-requirements.

Reply:
Please have a look at the recipes now and let me know if you think anything else must be changed.